CMakeLists.txt

```diff
@@ -178,10 +190,14 @@ if (${CMAKE_SYSTEM_NAME} STREQUAL "iOS")
   set(QFIELD_RESOURCE_FILES ${SHARE_PATH})
 endif()
 
-set (ENABLE_TESTS TRUE CACHE BOOL "Build unit tests?")
+set(ENABLE_TESTS CACHE BOOL "Build unit tests")
+IF(ANDROID)
+  set(ENABLE_TESTS FALSE)
+else()
+  set(ENABLE_TESTS TRUE)
```

Comment:
This will override any user defined (`CACHE`) value from line 193.
Either we leave it to the user to disable tests (via command line `-DENABLE_TESTS=FALSE`) or this is set before 193 and passed in as default value.

Reply:
Let's pass the variable in as default